src/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
```suggestion

```
it's forbidden to use anything else except box-sizing in *. Move text styles to `html` selector